content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -480,7 +480,7 @@ export class ASRouterAdminInner extends React.PureComponent {
     if (!this.state.providers) {
       return null;
     }
-    return (<p>Show messages from <select value={this.state.messageFilter} onChange={this.onChangeMessageFilter}>
+    return (<p>Show messages from <select value={this.state.messageFilter} onBlur={this.onChangeMessageFilter}>
```

Comment:
Travis automation has this test failing: https://github.com/mozilla/activity-stream/blob/5175b975c5121b561e1ea4945f10ba60855f05c6/test/unit/content-src/components/ASRouterAdmin.test.jsx#L167-L178

You can run all the tests locally with `npm test`

If you want to run just this one test linked above, you can edit the file temporarily adding a `.only` to become `it.only("should hide…")`